LocalWiki.php

```diff
@@ -32,6 +32,37 @@ function onBeforePageDisplay( OutputPage $out ) {
 		$wgJsonConfigs['Map.JsonConfig']['store'] = true;
 		$wgJsonConfigs['Tabular.JsonConfig']['store'] = true;
 		
+		break;
+  case 'gratisdatawiki':
+		$wgHooks['BeforePageDisplay'][] = 'onBeforePageDisplay';
+		
+		function onBeforePageDisplay( OutputPage $outputPage ) {
+			$out->addMeta( 'og:image:width', '1200' );
+			
+			// The title of wikibase entities will be the label, if available, or else the entity id (e.g. 'Q42').
+			$meta = $this->getOutput()->getProperty( 'wikibase-meta-tags' );
+			// This would remove description, social media tags, or any search engine optimization for diffs
+			if ( $this->isDiff() ) {
+				if ( isset( $meta['title'] ) ) {
+					$this->setDiffPageTitle( $outputPage, $meta['title'] );
+				}
+			}
+			
+			if ( isset( $meta['title'] ) ) {
+				$this->setHTMLTitle( $outputPage, $meta['title'] );
```

Comment:
Doesn't exist.

Reply:
> Doesn't exist.

@Universal-Omega See this https://github.com/wikimedia/Wikibase/blob/master/repo/includes/Actions/ViewEntityAction.php#L140

Reply:
Yeah but not in this context.

Reply:
LocalWiki.php is not $this in ViewEntityAction.php.

Reply:
> LocalWiki.php is not $this in ViewEntityAction.php.

Oh, what do we use then?